documentation/workflows.md

```diff
@@ -0,0 +1,74 @@
+# Working on the Project
+
+Better the be explicit/specific on topics than to wait for avoidable confusion due to ambiguity.
```

Comment:
```suggestion
Better to be explicit/specific on topics than to wait for avoidable confusion due to ambiguity.
```
The intention of this sentence is a little unclear if this document is intended to orient users to a specific workflow. What topics? Maybe some information about how to select an issue, what the tags mean on issues and how to claim something so that it is visible what is being worked on.